API.md

```diff
@@ -3015,7 +3017,7 @@ Each request object includes the following properties:
   different response. Contains `null` when no response has been set (e.g. when a request terminates
   prematurely when the client disconnects).
 - `preResponses` - same as `pre` but represented as the response object created by the pre method.
-- `query` - an object containing the query parameters.
+- `query` - an object containing the parsed [query parameters](https://nodejs.org/docs/latest/api/url.html#url_urlobject_query).
```

Comment:
I would use `might` instead of `may` but that is nitpicking, the rest seems good! ^^